src/architecture.md

```diff
@@ -0,0 +1,104 @@
+# Architecture
+
+## Diagram
+
+TBD
+
+## Main components
+
```

Comment:
May start with the fact that kubewarden is a custom resource and kubewarden-controller is the implementation of this resource. It starts 'policy-server' as a helper to provide the safe Wasm sandbox.

Reply:
Also a brief introduction to (Kubernetes) policies in general (policies are registered with k8s, multiple policies can be chained, policy passes only if all in the chain pass, etc.) Then link to https://kubernetes.io/docs/concepts/policy/